src/helm/benchmark/static/benchmarking.js

```diff
@@ -693,16 +687,16 @@ $(function () {
 
     // Paths (parallel arrays corresponding to `runSpecs`)
     const statsPaths = runSpecs.map((runSpec) => {
-      return statsJsonUrl(suite, runSpec.name);
+      return statsJsonUrl(getRunSuite(usingRelease, release, runsToRunSuites, runSpec.name), runSpec.name);
```

Comment:
`release` is undefined here; you either need to make it a global, or pass it into `renderRunsDetailed()`.

Reply:
Done.